src/edu/csus/ecs/pc2/core/imports/clics/FieldCompareRecord.java

```diff
@@ -0,0 +1,146 @@
+// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.imports.clics;
+
+import java.util.logging.Level;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.log.StaticLog;
+import edu.csus.ecs.pc2.core.report.ComparisonState;
+import edu.csus.ecs.pc2.core.report.FileComparisonUtilities;
+
+/**
+ * Information about a comparison of two fields.
```

Comment:
This statement about what the class represents is confusing.  It claims the class holds information about "two fields" -- but in fact there is just a _single_ "field name" associated with the class.  Is the intent that the class holds information about the _values_ in two_different _instances_ of a field with the same field _name_?  If so, the comment should state that.